### PR TITLE
ffmpeg: add pulseaudio build_option

### PR DESCRIPTION
**Pull request by ioraff**

### CODE REVIEW

**Comment by Johnnynator** [comment]
Try to make sure that you don't have any merge commits. `ffmpeg: make pulseaudio optional build option` should be the only commit in this PR

File: srcpkgs/ffmpeg/template [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # audacity also needs to be bumped when a new ffmpeg version bumps libavformat's soname!
 pkgname=ffmpeg
 version=4.1.1
-revision=1
+revision=2
```
Comment (Johnnynator) [inline srcpkgs/ffmpeg/template]: revbump is not needed, since the resulting package should have the same content as it has currently.

**Comment by maxice8** [comment]
optional build_option is redundant, all build_options are optional, they are options a user takes.

maybe

`ffmpeg: add pulseaudio build_option`

**Comment by maxice8** [comment]
Please rebase your changes to remove the merge commits.

**Pull request closed without merging**